lib/core/xskdev/xskdev.c

```diff
@@ -974,6 +974,17 @@ xskdev_socket_create(struct lport_cfg *c)
     if (fq_reserved(xi, umem->fq_size) < 0)
         CNE_ERR_GOTO(err, "Failed reserved fill of FQ\n");
 
+    do {
```

Comment:
What's with the "do {} while (0)", just to create a separate block for your local variables? Can you define your local variables at the top of the function or create a new function to do this?

Reply:
removed